### ISSUE TITLE
[LINT_BUG]: Multiple box::use() calls cause lint to fail

### ISSUE THREAD

**Issue by caldwellst** [Appsilon/box.linters#111]
### box.linters version

0.9.1

### Sample source code to lint

```r
box::use(dplyr)
box::use(stringr)

#' @export
x <- function() {
  dplyr$mutate(mtcars)
  stringr$str_detect("a", "a")
}
```

### Lint command used

```r
lintr::lint(
  code,
  linters = lintr::linters_with_defaults(defaults = box.linters::box_default_linters)
)
```

### Lint result

```
Line 1 [box_unused_att_pkg_linter] Attached package unused.
Line 2 [box_unused_att_pkg_linter] Attached package unused.
Line 6 [box_usage_linter] <package/module>$function does not exist.
Line 7 [box_usage_linter] <package/module>$function does not exist.
```

### Expected result

This should succeed. Linting is successful if we only import a single package.

```r
box::use(dplyr)

#' @export
x <- function() {
  dplyr$mutate(mtcars)
}
```

It is also successful with multiple imports if we follow the `box.linters::rhino_default_linters` style of adding `, ` after all imports.

```r
box::use(dplyr, )
box::use(stringr, )

#' @export
x <- function() {
  dplyr$mutate(mtcars)
  stringr$str_detect("a", "a")
}
```

And if we only add `,` without the space, it is also successful, which leads me to suspect it has something to do with expecting comma separation that allows the linter to successfully find multiple imports.

**Comment by radbasa** [comment]
This is interesting.

**Comment by radbasa** [comment]
While linting `box::use(packages)` use a different set of functions from linting `box::use(path/modules)`, they both follow the same algorithm.

We, at Appsilon, as standard practice use one `box::use()` call for packages, and a separate `box::use()` call for modules. We did not take into consideration having multiple `box::use()` calls. We will include this feature request in future releases.

Thanks.

**Comment by radbasa** [comment]
@caldwellst We just released `0.10.0` on CRAN. It comes with a fix for this issue.
